streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -621,6 +621,7 @@ private void processData() {
         driver.process(TOPIC, "1", "D");
         driver.process(TOPIC, "3", "E");
         driver.process(TOPIC, "3", "F");
+        driver.process(TOPIC, "3", null);
```

Comment:
This should take care of the reduce and aggregates called from test cases ( which call it ). How to run them locally ? Please advise or point to any documentation on ho w to run it locally. Thank you.

Reply:
Yes this sounds good.

IN case you want to run it locally, feel free to read the README.md file, which includes cmd for running a single test.